src/images/optimisation/index.ts

```diff
@@ -0,0 +1,31 @@
+import ImageBlobReduce, { Options } from 'image-blob-reduce';
```

Comment:
`image-blob-reduce` is a wrapper around the [pica library](https://github.com/nodeca/pica). Under the hood, it uses the [`mks2013`](https://johncostella.com/magic/) algorithm to handle image optimisation, and handles resizing via the use of a `canvas` element.